module1/lessons/inheritance.md

```diff
@@ -272,14 +335,26 @@ class Employee
 end  
 ```
 
+Note that in `base_salary` and `bonus`, we are using `self.class` to refer to the class that called the method. This is important because it allows us to use the `raise` keyword to raise an error if the method is called on a subclass that does not have the necessary instance variables defined.
+</section>
+</section>
+
+
```

Comment:
I think adding a note about that would be good, that other languages also implement inheritance and how they do it may be different, but the core idea is the same.

Reply:
Aye good callout, I agree!  Thanks for the suggestion, I'll get that added.